### PR TITLE
Removed checkov skips  @melvabout

### PR DESCRIPTION
**Pull request by melvabout**

### CODE REVIEW

File: service_catalog_portfolios/storage/dynamo/template.yaml [hunk 1]
```diff
@@ -315,6 +315,10 @@ Resources:
                   - "cloudwatch:SetAlarmState"
                   - "cloudwatch:DeleAlaterms"
```
Comment (dangital) [inline service_catalog_portfolios/storage/dynamo/template.yaml:316]: Not related to these changes but DeleAlaterms > DeleteAlarms

File: service_catalog_portfolios/compute/lambda/sam-cfn-template.yaml [hunk 2]
```diff
@@ -205,6 +211,7 @@ Resources:
     Type: AWS::SQS::Queue
     Properties: 
       QueueName: !Sub "${AWS::StackName}"
+      KmsMasterKeyId: !Ref KmsKeyId
```
Comment (dangital) [inline service_catalog_portfolios/compute/lambda/sam-cfn-template.yaml:214]: I think this also needs to be KmsKeyArn

Comment (melvabout) [reply service_catalog_portfolios/compute/lambda/sam-cfn-template.yaml]: KMs key creation move to within the template. Needed specific condition for use with logs

File: service_catalog_portfolios/storage/dynamo/template.yaml [hunk 3]
```diff
@@ -313,8 +313,12 @@ Resources:
                   - "cloudwatch:DescribeAlarms"
                   - "cloudwatch:GetMetricStatistics"
                   - "cloudwatch:SetAlarmState"
-                  - "cloudwatch:DeleAlaterms"
+                  - "cloudwatch:DeleteAlaterms"
```
Comment (dangital) [inline service_catalog_portfolios/storage/dynamo/template.yaml:316]: Still a tyop

**Review by dangital — APPROVED** [review]

**Pull request merged**